psamm/datasource/sbml.py

```diff
@@ -868,5 +894,16 @@ def write_model(self, file, model):
                 self._sbml_tag('constant'): 'true'
             })
 
+        # list all gene products
+        listallgenes = ET.SubElement(model_tag, _tag(
+            'listOfGeneProducts', FBC_V2))
```

Comment:
Generally variable names should be `snake_case` instead of running all words together.